packages/seal/src/Schema/SchemaHelper.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace CmsIg\Seal\Schema;
+
+class SchemaHelper
```

Comment:
I would keep this internal:

```suggestion
/**
 * @internal
 */
class SchemaHelper
```

But also see later comment I think I would not even use this.

Reply:
Deleted :)